src/__tests__/users.test.ts

```diff
@@ -0,0 +1,38 @@
+import request from "supertest";
+import app from "../setUpServer";
+import { setupDB } from "./tests_setup";
+
+setupDB();
+
+// describe("POST /users/login", () => {
```

Comment:
I'd like to finish implementing the request that returns a 200 OK upon being provided valid creds here